src/nomad_material_processing/vapor_deposition.py

```diff
@@ -246,7 +241,15 @@ class Substrate(ArchiveSection):
     distance_to_source = Quantity(
         type=float,
         unit="meter",
-        shape=["*"],
+        #shape=["*"],    ############################### why is it an array? this is a process parameter equal inside the chamber
```

Comment:
It's an array because there are usually multiple sources. As we discussed in the last meeting, this should eventually be replaced by recording the position of all components in the chamber.

Reply:
So if we have multiple substrate, are all the sources duplicated inside each Substrate class? ?

Reply:
The sources are stored in a separate section of the PVDStep. This is just keeping track of the distance from each substrate to all the sources.

Reply:
But we definitely need a description here for now. I will add that.

Reply:
Yes, I know sources are elsewhere. I meant this quantity would be duplicated in each Substarte?

Reply:
ok, maybe I get it. This array of distances would vary as their distances are not constant with respect to each substrate.